release_notes/ocp_3_7_release_notes.adoc

```diff
@@ -1172,6 +1172,17 @@ Cluster Metrics] for more information.
 [[ocp-37-developer-experience]]
 === Developer Experience
 
+[[ocp-37-aws-service-broker]]
+==== AWS Service Broker
+
+Users can seamlessly configure, deploy, and scale AWS services like Amazon RDS,
+Amazon Aurora, Amazon Athena, Amazon Route 53, and AWS Elastic Load Balancing
+directly within the {product-title} console. For installation instructions, see
+link:https://github.com/awslabs/aws-servicebroker-documentation/wiki[AWS Service
```

Comment:
The link points to: https://github.com/awslabs/aws-servicebroker/

Do you think it should go directly to the new repo?